packages/multichain/src/scope/validation.test.ts

```diff
@@ -0,0 +1,180 @@
+import type { ExternalScopeObject } from './types';
+import { isValidScope, validateScopes } from './validation';
+
+const validScopeString = 'eip155:1';
+const validScopeObject: ExternalScopeObject = {
+  methods: [],
+  notifications: [],
+};
+
+describe('Scope Validation', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
```

Comment:
This has been happening automatically since https://github.com/MetaMask/core/pull/4417, so it is not needed:

``` suggestion
```

Reply:
https://github.com/MetaMask/core/pull/4784/commits/18d1112b95e756334fbf06d7dd5b9ec24a8783f7